docs/reference/deploy/master-config-reference.rst

```diff
@@ -230,6 +230,13 @@ customize the UI.
 Specifies the path to a user-provided logo to be shown in the UI. Ensure the path is accessible and
 reachable by the master service. The logo file should be a valid image format, with SVG recommended.
 
+Logo path is defined in four variations:
```

Comment:
In reference to our conversation [here](https://hpe-aiatscale.slack.com/archives/CLCE8D998/p1724175767150509), should we mention in these (and the release note) docs that all four path variations are required when specifying the `logo_path` (once this requirement is enforced in the backend)?

Reply:
> docs that all four path variations are required

I added this to the config's Validate logic so the users would get error on it and should be more clear but yeah goo idea to include it here